src/Console/Command/InspectSymbolCommand.php

```diff
@@ -12,6 +10,8 @@
  * file that was distributed with this source code.
  */
 
+declare(strict_types=1);
```

Comment:
for this specific project, makes more sense to keep the declare strict above to reduce the diff